### DISCUSSION TITLE
Label on onboarding MR?

### DISCUSSION THREAD

**Discussion by twarner-sofi** [renovatebot/renovate#9517] [Request Help] [answered]
I am running selfhosted 24.113.0, pointed at gitlab.com

I have `labels: ['renovate']` in my bot config.js, and it adds the label to dep update MRs but not to the initial onboarding MRs. 

Is it possible to have it add the label to onboarding MRs?

**Comment by rarkins** [comment, marked as answer]
Currently Renovate doesn't add labels to onboarding PRs, but if you'd like to submit a PR adding the functionality then you'd be welcome
